viral/representational_drift.py

```
@@ -0,0 +1,828 @@
+    plt.tight_layout()
+    plt.suptitle(genotype)
+
+
+if __name__ == "__main__":
+    # for genotype in ["Oligo-BACE1-KO", "NLGF", "WT", "Neuronal-BACE1-KO"]:
+    #     plot_overall_scores(genotype)
+    # for genotype in ["Oligo-BACE1-KO", "NLGF", "WT", "Neuronal-BACE1-KO"]:
+    #     plot_landmark_drift_correlation_results(genotype, None)
+    main()
+
+    1 / 0
```

Ruff (0.14.4)

Found useless expression. Either assign it to a variable or remove it.

(B018)